src/fu-util.c

```diff
@@ -2441,6 +2464,12 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: firmware approved by the admin */
 		     _("Sets the list of approved firmware."),
 		     fu_util_set_approved_firmware);
+	fu_util_cmd_array_add (cmd_array,
+		     "self-sign",
+		     "TEXT",
+		     /* TRANSLATORS: command description */
+		     _("Sign data using the client certificate"),
+		     fu_util_self_sign);
```

Comment:
Other than for testing the signing mechanism why put this in fwupdmgr?  I would think it makes more sense to have a `--signed` option and automatically sign relevant payloads it makes sense (or maybe even `--not-signed`) depending on the specific use case.

Reply:
It was just for testing, but I agree it's a bit worthless if you can do the same kind of debugging in fwupdtool. I'll remove it, and add something like `--signed` for report uploading. If I get time today I'm going to look at the LVFS side to make sure all the signing stuff implemented as a client side cert works.